examples/gallery/maps/choropleth_map.py

```diff
@@ -20,7 +20,7 @@
 
 # Read the example dataset provided by geodatasets.
 gdf = gpd.read_file(geodatasets.get_path("geoda airbnb"))
-print(gdf)
+gdf
```

Comment:
```suggestion
gdf.head()
```

Reply:
Maybe keep the `.head()` to show only the first rows, but use it with `print`. Then the output is adjusted and the wide table fits on the page.

Reply:
Good suggestion. Done in 7e2cace77290ac59393d8f8c56142efcacf7a472.